### PR TITLE
Add node 16

### PR DESCRIPTION
**Pull request by jrpz**
Adds a docker for node 16

### CODE REVIEW

**Review by ivorisoutdoors — CHANGES_REQUESTED** [review]
@jrpz can you take a look at my PR #17 and match what I did with those images here?

Also make sure we add this to GitHub Action to get this building

**Comment by jrpz** [comment]
@mloberg sure thing!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,8 +20,9 @@ jobs:
           - 10-stretch-slim
           - 12-stretch-slim
           - 14-stretch-slim
+          - 16-stretch-slim
         include:
-          - tag: 14-stretch-slim
+          - tag: 16-stretch-slim
```
Comment (ivorisoutdoors) [inline .github/workflows/build.yml]: When we pull the _latest_ tag or don't define a tag, this will now be using Node 16. Is that expected?

**Review by ivorisoutdoors — DISMISSED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (jrpz) [reply .github/workflows/build.yml]: Yes, is that okay?  Do you see any reason to stick with `14` as the latest?

Comment (ivorisoutdoors) [reply .github/workflows/build.yml]: Wanted to make sure you understood this change. I think it's fine to bump as the latest

Comment (jrpz) [reply .github/workflows/build.yml]: On second thought, I'm reverting this back to `14`.  `16` should be tested in the wild before committing it to `latest`.  I'm sure `images` is the only service using this image and we specify the tag to 12... but best to be safe.

**Review by ivorisoutdoors — APPROVED** [review]

**Pull request merged**